app/views/posts/index.html.erb

```diff
@@ -1,11 +1,12 @@
 <%= render "shared/user_box", user: @user %>
 
-<% @posts.each do |post| %>
+<%# I'm sure that I solved N+1 problem when fetching all posts and their comments for a user, I'm using @posts.includes comments: :author) instead of @posts in my controllers %>
+<% @posts.includes(comments: :author).each do |post| %>
```

Comment:
- [Optional] Views should contain only presentation logic and not querying logic. This was supposed to be done in the index action of the Posts Controller.